docs/schema.md

```diff
@@ -241,6 +245,39 @@ Databases are encouraged not to include those in the first place. (The goal is
 to balance flexibility of presentation with not exposing vulnerability database
 display sites to unnecessary vulnerabilities.)
 
+## severity field
+
+```json
+{
+	"severity": [ {
+		"type": string,
+		"score": string
+	}
+} ]
```

Comment:
The } and ] should be swapped here. To format this consistently, the ] should go after the } on line 255 instead.

Reply:
Oops, thank you! Fixing